src/libs/dateFormat/index.test.ts

```diff
@@ -0,0 +1,35 @@
+import { dateFormat } from '.';
+
+describe('dateFormat function', () => {
+  it('formats the date correctly according to the given format', () => {
+    const date = '2023-01-01T00:00:00.000Z';
+    const formatString = 'yyyy-MM-dd';
+    const result = dateFormat(date, formatString);
+    expect(result).toBe('2023-01-01');
+  });
+
```

Comment:
実際のユースケースとして`format = 'yyyy/MM/dd HH:mm:ss'`があるなら，秒数までフォーマットできるかどうか試したやつがテスト項目に欲しくない？

> export const DateLabel: React.FC<Props> = ({
>   date,
>   format = 'yyyy/MM/dd HH:mm:ss',
>   className,
> }) => (
```suggestion

  it('formats the date (up to seconds) correctly according to the given format', () => {
    const date = '2023-01-01T00:00:00.000Z';
    const formatString = 'yyyy/MM/dd HH:mm:ss';
    const result = dateFormat(date, formatString);
    expect(result).toBe('2023/01/01 00:00:00');
  });
```

Reply:
おっしゃる通り